.github/workflows/run_tests.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
 
     steps:
-      - name: Checkout harmony-gdal-adapter repository
+      - name: Checkout earthdata-varinfo repository
```

Comment:
Whoaa what does this do?

Reply:
This change just fixes where I copied over the CI/CD from the HGA repo and missed changing a reference from HGA to earthdata-varinfo. In particular, you can see the name of the step of this workflow change (see the second step in the runs [before](https://github.com/nasa/earthdata-varinfo/actions/runs/6194333475/job/16817159230) vs [after](https://github.com/nasa/earthdata-varinfo/actions/runs/6194450426/job/16817463275)).

Not sure if you've played around much with GitHub Actions before. If not, you can get your CI/CD to run in GitHub by defining it in YAML files that sit in `.github/workflows`. (I _really_ like having it all built into GitHub, and having the CI/CD under version control. Plus, they are just fun to play around with, even if debugging can be a bit of a pain)

Reply:
(I guess to clarify, this isn't a functional change, just essentially changing a label for a step in the workflow)